Create Fina_e_os_mapas_text-de-CH.md

sfkeller opened the pull request:
Übersetzt von Stefan Keller, sfkeller@gmail.com (Schweiz)

psanxiao commented:
Astrid (@astridx), Stefan (@sfkeller) , first of all, thank you so much for your contributions. It's fantastic to have the book translated into several languages.

I don't know much about German, and I'm not familiar with the differences between the DE and de-CH versions. In the case of Spanish, for example, we tried to have a single translation, choosing words and expressions that are more neutral and suitable for both Spain and Latin America.

Do you think it would be possible to do the same for German, or is it necessary to have different translations?

sfkeller commented:
Hi Pablo (@psanxiao), hallo Astrid (@astridx),

First of all, I'd like to thank you for the beautiful booklet, which I also found via Weekly OSM. I've been dreaming of such a booklet for a long time, known as Pixi books in Germany and Switzerland. 

> Do you think it would be possible to do the same for German, or is it necessary to have different translations?

I thought about before this but ended up to say "no", because the booklet is aimed at children who would be confused. It's perhaps like the words "Abuela" vs. "Nana". And above all, there is no "ß" in Swiss high german (it's "ss"). 

Yours Stefan

P.S. Just nitpicking: Regarding the file ending you mention "DE" and "de-CH" (and ES) I deliberately used IETF language codes, as is common in localisation. IETF language codes are based on ISO 639-1 (language, lowercase) + ISO 3166-1 Alpha-2 (country code, uppercase). Hence: spanish → es-ES, galician → gl-ES, catalan → ca-ES, basque→ eu-ES, german → de-DE, Swiss high german → de-CH, Swiss French → fr-CH...

psanxiao commented:
Many thanks to both of you for your points of view. It is now clear to me that it is better to keep separate versions.

There is a small manual step to integrate the translations, as we need to ensure that the texts fit well on the pages and make minor adjustments, but it’s nothing serious.

As for the observation about using IETF codes, I agree. I will gradually update the other translation files to use them as well. Thanks for the suggestion.

sfkeller commented:
Hi Pablo (@psanxiao), 

Thx, Pablo, for merging! Astrid (@astridx) it was great to discuss this. I invite you both to have a cup of coffee here in Rapperswil, the "most beautiful campus in Switzerland" if you are nearby. It's at the other end of Lake Zurich.

Pablo answered above:

> There is a small manual step to integrate the translations, as we need to ensure that the texts fit well on the pages ...

To understand you correctly: Will there be another SVG in "Code" ?

And another question about the font size: What is the target format for printing this beautiful booklet? I ask because I plan to hanld an hard print it as a booklet similar to the german Pixi series, which are 10 x 15 cm, for example. But even if it's A4 landscape, the font size in the current SVG example seem to me rather small. So I would suggest to set the font size to be as large and bold as possible.

psanxiao commented:
> To understand you correctly: Will there be another SVG in "Code" ?
> 
> And another question about the font size: What is the target format for printing this beautiful booklet? I ask because I plan to hanld an hard print it as a booklet similar to the german Pixi series, which are 10 x 15 cm, for example. But even if it's A4 landscape, the font size in the current SVG example seem to me rather small. So I would suggest to set the font size to be as large and bold as possible.

The book was designed for a digital format, to be read on a screen (computer or tablet). From the original version, in Galician, we did make a small printed edition to distribute in libraries and schools here in Galicia.

That printed version was formatted slightly differently, in A5 format. Here you can see a video of how it turned out: [link](https://www.instagram.com/p/DEnM8_0NTzv/?img_index=2)

I have the A5 version as a print-ready PDF in Galician. If you're very interested, I could ask about the editable version.

From the original printed version in Galician, if you give me a postal address, I can send you a copy. You can write to me privately at psanxiao@gmail.com.

sfkeller commented:
Pablo wrote:

> That printed version was formatted slightly differently, in A5 format. Here you can see a video of how it turned out (...)

Oh; nice video. And what a luxurious hardcover book. 

> The book was designed for a digital format, to be read on a screen (computer or tablet). (...)

I see. That makes sense. 

So you generate an SVG (16:9) from all the translated versions (I'm keen on the de-CH version, of course 
:wink:)?

psanxiao commented:
> So you generate an SVG (16:9) from all the translated versions (I'm keen on the de-CH version, of course
> 😉)?

Yeah, this is the idea. I can upload the translated svg file when is ready.

psanxiao commented:
Hello again @sfkeller 

We have finally been able to generate the SVG for the Swiss German translation.

I'm sharing it with you through this [Google Drive link](https://drive.google.com/file/d/1091KL0Pq8kT2o9MNwFDYE9FgAJudVC5K/view?usp=sharing). Since these files are quite large and we already have several translations, we prefer not to overload the repository.

If possible, please check whether the line breaks in the sentences are correct or if any of them make reading difficult. Also, let us know if you spot any errors so we can fix them before generating the PDF and uploading it to the website.

Thank you very much!

sfkeller commented:
Hi @psanxiao 

Thank you so much. I realize now what layout work is needed to integrate the text into the illustrations!

I've changed some word to correct the content and added typographical corrections in this pull request https://github.com/psanxiao/finaeosmapas/pull/9 .

I've several remarks (besides the corrected words and typographic corrections):

1. Page 2 Credits: I have taken the liberty of mentioning myself as a translator. "Übersetzung in die deutsche Sprache (Schweiz): Stefan F. Keller". This would apply to all translation and their authors. Pls. feel free to reject/delete this for any reason.
2. Page 2 Credits: I also have move the sentence "Weitere Informationen zu dieser Lizenz findest du hier: https://creativecommons.org/licenses/by-sa/4.0/deed.de" before the credits to "openstreetmap.org/copyright (which would also sensibel for the source).
3. On the following pages, the text is cut off on the right: 12, 16, 18, 19, 20.
4. Page 20 / last page: The URL and QR code should not be www.openstreetmap.com, but www.openstreetmap.org. This also applies to the original. And before, there should probably be www.ghandalf.org.

-- Stefan

psanxiao commented:
Hello Stefan,

After a few busy days, I'm getting back to the translations.

I've reviewed your changes and adjusted the SVG. You should see the updated SVG in the link I sent you. Here is also the link to the PDF: https://drive.google.com/file/d/1SwLZN8YqegX6EjL0hCA4qmTLxRPQBShA/view?usp=sharing

If everything looks good to you, I'll upload the PDF to the website (finaeosmapas.ghandalf.org) so it will be available for download.

Thanks a lot!

das-g commented:
Direkte Rede durch den Geviertstrich (—; engl. en-dash) einzuleiten oder abzugrenzen ist im Deutschen (inkl. in dessen schweizerischen Variante) unüblich. Stattdessen werden [Anführungszeichen](https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen) verwendet, siehe insb. den [Wikipedia-Unterabschnitt zu typografischen Anführungszeichen in der Schweiz](https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen#Schweiz,_Liechtenstein,_Frankreich).

sfkeller commented:
Also typografisch möchte ich denen nicht dreinreden – nur wenn der Text nicht lesbar wäre…

LG Stefan

From: Raphael Das Gupta ***@***.***>
Sent: Wednesday, March 12, 2025 4:31 PM
To: psanxiao/finaeosmapas ***@***.***>
Cc: Stefan F. Keller ***@***.***>; Mention ***@***.***>
Subject: Re: [psanxiao/finaeosmapas] Create Fina_e_os_mapas_text-de-CH.md (PR #2)


Direkte Rede durch den Geviertstrich (—; engl. en-dash) einzuleiten oder abzugrenzen ist im Deutschen (inkl. in dessen schweizerischen Variante) unüblich. Stattdessen werden Anführungszeichen<https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen> verwendet, siehe insb. den Wikipedia-Unterabschnitt zu typografischen Anführungszeichen in der Schweiz<https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen#Schweiz,_Liechtenstein,_Frankreich>.

—
Reply to this email directly, view it on GitHub<https://github.com/psanxiao/finaeosmapas/pull/2#issuecomment-2718278128>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAAKSTGVTKUBHE6QP7RBNFD2UBHJTAVCNFSM6AAAAABWZLQCSOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMJYGI3TQMJSHA>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
[das-g]das-g left a comment (psanxiao/finaeosmapas#2)<https://github.com/psanxiao/finaeosmapas/pull/2#issuecomment-2718278128>

Direkte Rede durch den Geviertstrich (—; engl. en-dash) einzuleiten oder abzugrenzen ist im Deutschen (inkl. in dessen schweizerischen Variante) unüblich. Stattdessen werden Anführungszeichen<https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen> verwendet, siehe insb. den Wikipedia-Unterabschnitt zu typografischen Anführungszeichen in der Schweiz<https://de.wikipedia.org/wiki/Anf%C3%BChrungszeichen#Schweiz,_Liechtenstein,_Frankreich>.

—
Reply to this email directly, view it on GitHub<https://github.com/psanxiao/finaeosmapas/pull/2#issuecomment-2718278128>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAAKSTGVTKUBHE6QP7RBNFD2UBHJTAVCNFSM6AAAAABWZLQCSOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMJYGI3TQMJSHA>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

sfkeller commented:
Hi @psanxiao,

You wrote:
> If everything looks good to you, I'll upload the PDF to the website (finaeosmapas.ghandalf.org) so it will be available for download.

Looks good for me. Thank you for your work.
You are welcome to publish the PDF on the website.
-- Stefan

psanxiao commented:
The PDF is already published on the website: https://finaeosmapas.ghandalf.org/

Thank you very much for the review. Regarding the use of dashes or quotation marks in the dialogues, the German version used quotation marks, as the text file came this way. If it's also better to use quotation marks for Swiss German, I have no objection to modifying it.

Right now, I have several more translations pending preparation, and I need to work on them. That's why I uploaded it this way, but we can revisit it later and update the version.

sfkeller commented:
> If it's also better to use quotation marks for Swiss German, I have no objection to modifying it.

Ok. So I'll make another pull request for an update. That will take some days.

Then I have another important suggestion for improvement: It is not called Swiss German but Swiss High German. Swiss German is something rather different (see https://en.wikipedia.org/wiki/Swiss_Standard_German ). 

That's why it should be called Swiss High German at https://finaeosmapas.ghandalf.org/ under “Downloads”.

In addition, in standard German, maps are written in capital letters, so instead of “Fina und die karten” it should be “Fina und die Karten” and also “Fina entdeckt die Kartografie” (with “K” capitalized).

![image](https://github.com/user-attachments/assets/ee916752-5b44-4621-aabf-a448a61a73af)

psanxiao commented:
> Then I have another important suggestion for improvement: It is not called Swiss German but Swiss High German. Swiss German is something rather different (see https://en.wikipedia.org/wiki/Swiss_Standard_German ).
> 
> That's why it should be called Swiss High German at https://finaeosmapas.ghandalf.org/ under “Downloads”.
> 
> In addition, in standard German, maps are written in capital letters, so instead of “Fina und die karten” it should be “Fina und die Karten” and also “Fina entdeckt die Kartografie” (with “K” capitalized).

Good to know. Thanks for the notes. It should be correct now on the website.

das-g commented:
> maps are written in capital letters

_Nouns_ (not just proper nouns, but _all_ nouns) in German are written with initial capital letter, not just map-related ones. :smile:
